Express/model.js

```diff
@@ -0,0 +1,12 @@
+const format = require("pg-format");
+const db = require("../db/connection");
+
+const fetchAllTopics = async () => {
+    const {rows} = await db.query(`select * from topics`);
+    if (rows.length === 0) {
+       return Promise.reject({status: 404, msg: "Not found"});
```

Comment:
How can you test for this error? If we are implementing some error handling we need to also be able to test for it.